SQL/database_changelog.md

```diff
@@ -2,7 +2,7 @@ Any time you make a change to the schema files, remember to increment the databa
 
 Make sure to also update `DB_MAJOR_VERSION` and `DB_MINOR_VERSION`, which can be found in `code/__DEFINES/subsystem.dm`.
 
-The latest database version is 5.28; The query to update the schema revision table is:
+The latest database version is 5.31; The query to update the schema revision table is:
```

Comment:
the define is wrong too, see <https://github.com/tgstation/tgstation/pull/91420>

Reply:
AJHHHHHHHHHHH

Reply:
anyway yes but also the .dm file is wrong too, the minor db version define was not ever updated to 31